index.d.ts

```diff
@@ -68,7 +69,33 @@ declare namespace Eris {
   type MessageContent = string | AdvancedMessageContent;
   type MFALevel = 0 | 1;
   type PossiblyUncachedMessage = Message | { channel: TextableChannel | { id: string; guild?: Uncached }; guildID?: string; id: string };
-  type InteractionType = 1 | 2;
+
+  // Interaction
+  type InteractionType = 1 | 2 | 3;
+  type InteractionResponseType = 1 | 4 | 5 | 6 | 7;
+  type SlashCommandOptionType = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
+
+  type InteractionDataOptions = {
+    name: string;
+    type: SlashCommandOptionType;
+    value?: SlashCommandOptionType;
+    options?: InteractionDataOptions[];
+  };
+
+  type InteractionOptions = {
+    type: InteractionResponseType;
+    data?: InteractionContent;
+  };
+
+  type InteractionContent = {
+    allowedMentions?: AllowedMentions;
+    content?: string;
+    embeds?: EmbedOptions;
```

Comment:
```suggestion
    embeds?: EmbedOptions[];
```
Should be an array of [embeds](https://discord.com/developers/docs/resources/channel#embed-object) instead

[Here](https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-application-command-callback-data-structure)

Reply:
yep i missed this